### PR TITLE
[Seedless-Onboarding] Add tracking

### PR DESCRIPTION
**Pull request by schmanu**
## What it solves
Tracking for the MPC Wallet

## How this PR fixes it
Adds tracking for the following events:
- Continue to creation button with connected Wallet
- Continue with Google button
- Account recovery started
- Recover account using password
- Set or change password
- Enable MFA for account

## How to test it
Test those various events.

## Checklist
* [x] I've tested the branch on mobile 📱
* [x] I've documented how it affects the analytics (if at all) 📊
* [x] I've written a unit/e2e test for it (if applicable) 🧑‍💻

### CODE REVIEW

File: src/components/welcome/WelcomeLogin/index.tsx [hunk 1]
```diff
@@ -5,15 +5,13 @@ import SafeLogo from '@/public/images/logo-text.svg'
...
-import { CREATE_SAFE_EVENTS, LOAD_SAFE_EVENTS } from '@/services/analytics/events/createLoadSafe'
+import { LOAD_SAFE_EVENTS } from '@/services/analytics/events/createLoadSafe'
 import Track from '@/components/common/Track'
-import { trackEvent } from '@/services/analytics'
 
 const WelcomeLogin = () => {
   const router = useRouter()
 
   const continueToCreation = () => {
-    trackEvent(CREATE_SAFE_EVENTS.CREATE_BUTTON)
```
Comment (usame-algan) [inline src/components/welcome/WelcomeLogin/index.tsx:16]: Should we still use this event or remove it altogether because there is overlap now?

Comment (usame-algan) [reply src/components/welcome/WelcomeLogin/index.tsx:16]: Also removing it here means we are not tracking it anymore if a user presses "Connect a wallet" on the welcome screen and gets redirected to the safe creation.

File: src/components/common/ConnectWallet/MPCLogin.tsx [hunk 2]
```diff
@@ -34,44 +37,48 @@ const MPCLogin = ({ onLogin }: { onLogin?: () => void }) => {
...
+                  </Typography>
+                  <Typography variant="body2">{userInfo.email}</Typography>
+                </div>
+                <SvgIcon component={GoogleLogo} inheritViewBox fontSize="medium" sx={{ marginLeft: 'auto' }} />
+              </Box>
+            </Button>
+          </Track>
+        </>
+      ) : (
+        <Track {...MPC_WALLET_EVENTS.CONNECT_GOOGLE}>
```
Comment (usame-algan) [inline src/components/common/ConnectWallet/MPCLogin.tsx:68]: Wrapping the button with the `Track` component made it look smaller compared to the other button. We could make it a div or add 100% width on it.

Screenshot 2023-10-13 at 09 25 49

File: src/components/welcome/WelcomeLogin/index.tsx (hunk 1 above)
Comment (schmanu) [reply src/components/welcome/WelcomeLogin/index.tsx:16]: I renamed the event to OPEN_SAFE_CREATION as it's action label is "Open stepper" and emit it here again.

**Comment by katspaugh** [comment]
IMHO the wallet type should be a user-level property, not event level. Please reseatch how to do this with GTM.

**Review by usame-algan — APPROVED** [review]

**Pull request merged**